django_ipam/api/views.py

```diff
@@ -0,0 +1,16 @@
+import swapper
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+Subnet = swapper.load_model('django_ipam', 'Subnet')
+
+
+@api_view(['GET'])
```

Comment:
use `POST`, `GET` should be used only for idempotent requests (if you don't know what this means look for "idempotence").